source/website/src/views/Step3.vue

```diff
@@ -396,6 +398,25 @@ SPDX-License-Identifier: Apache-2.0
           };
           reader.readAsText(files[0]);
       },
+      resolveKeys(data) {
```

Comment:
add a comment to describe what this function does

Reply:
Resolved